e2e/pageobjects/ide/ProjectTree.ts

```diff
@@ -26,7 +26,7 @@ export class ProjectTree {
         @inject(CLASSES.Editor) private readonly editor: Editor) { }
 
     async openProjectTreeContainer(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
-        const selectedExplorerButtonLocator: By = By.xpath(Ide.SELECTED_EXPLORER_BUTTON_XPATH);
+        const selectedExplorerButtonLocator: By = By.css(Ide.SELECTED_EXPLORER_BUTTON_XPATH);
```

Comment:
```
By.css(Ide.SELECTED_EXPLORER_BUTTON_XPATH);
```
Looks a bit weird "By.css" and "SELECTED_EXPLORER_BUTTON_**XPATH**"

Reply:
replaced on SELECTED_EXPLORER_BUTTON_CSS